### PR TITLE
Integration Test for ActionRepository

### PR DESCRIPTION
**Pull request by maksontakson**

### CODE REVIEW

File: ActionStorage-master/src/test/java/app/just/actionstorage/repository/AbstractTest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import org.springframework.test.context.DynamicPropertySource;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+public class AbstractTest {
+  private static final PostgreSQLContainer CONTAINER = (PostgreSQLContainer) new PostgreSQLContainer<>("postgres:11.7")
+      .withReuse(true);
+
+  @DynamicPropertySource
+  public static void overrideProps(DynamicPropertyRegistry registry) {
+    registry.add("spring.datasource.url", CONTAINER::getJdbcUrl);
```
Comment (palyvtop) [inline ActionStorage-master/src/test/java/app/just/actionstorage/repository/AbstractTest.java]: Check driver for db

**Review by palyvtop — CHANGES_REQUESTED** [review]

File: ActionStorage-master/src/test/java/app/just/actionstorage/repository/AbstractTest.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    registry.add("spring.datasource.username", CONTAINER::getUsername);
+    registry.add("spring.datasource.password", CONTAINER::getPassword);
```
Comment (palyvtop) [inline ActionStorage-master/src/test/java/app/just/actionstorage/repository/AbstractTest.java]: You can move this property in application.yaml

**Review by palyvtop — CHANGES_REQUESTED** [review]

File: ActionStorage-master/src/test/java/app/just/actionstorage/repository/AbstractTest.java (hunk 2 above)
Comment (maksontakson) [reply ActionStorage-master/src/test/java/app/just/actionstorage/repository/AbstractTest.java]: I can't then inject it to my class beacuse of static modifier and this class is not a Bean of Spring.

**Comment by maksontakson** [comment]
Sorry, i had to add new commit while resolving conflicts here

**Review by palyvtop — APPROVED** [review]

**Pull request merged**